tests/Feature/HasBinaryUuidTest.php

```diff
@@ -120,6 +120,20 @@ public function it_prevents_double_encoding()
         $this->assertEquals($encodeUuid, $decodedUuid);
     }
 
+    /** @test */
+    public function it_generates_valid_routes()
+    {
+        $uuid = Uuid::uuid1();
+        $model = $this->createModel($uuid);
+
+        app('router')->get('uuid-test/{model}')->name('uuid-test');
+
+        $expected = "http://localhost/uuid-test/$uuid";
```

Comment:
Is hard-coding localhost the best idea? Maybe it would be better to assert like so?

```
$this->assertContains($actual, "/uuid-test/{$uuid}");
```